src/test/kotlin/org/openrewrite/java/quarkus/quarkus2/package-info.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.quarkus.quarkus2;
```

Comment:
again, not sure if this was a good idea-- happy to remove this from it's own independent package and leave it in the flat structure. Was just thinking about whether quarkus 1.x -> 1.x-latest should be separated from quarkus 1.x -> 2.x recipes.